packages/by-name/cli-release/package.nix

```diff
@@ -3,8 +3,8 @@
 
 { lib
 , contrast
-, genpolicy-msft
-, genpolicy ? genpolicy-msft
```

Comment:
@msanft could you please add a commit that exposed the genpolicy version used in the cli as `passthru.genpolicy`, so we can refer to the `settings`/`rules` in `cli-release` via `contrast.genplicy`? That will prevent us from mismatching.

Reply:
done.